### PR TITLE
CP-133: Error out if the HeaderData doesn't exist

### PR DESCRIPTION
**Pull request by RobLoach**
https://kalamuna.atlassian.net/browse/CP-133

### CODE REVIEW

File: smart_content_cdn.install [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
+ * @file
+ * Install hook implementations for Smart Content CDN module.
+ */
+
+/**
+ * Implementation of hook_requirements().
+ *
+ * During install of the smart_content_cdn module, make sure the library is installed.
```
Comment (RobLoach) [inline smart_content_cdn.install]:
``` suggestion
 * Implements hook_requirements().
```

File: smart_content_cdn.install [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+function smart_content_cdn_requirements($phase) {
+  $requirements = [];
+
+  if ($phase == 'install') {
+    // Make sure the HeaderData class is available.
+    if (!class_exists('Pantheon\EI\HeaderData')) {
+      $requirements['smart_content_cdn'] = [
+        'title' => t('Smart Content CDN'),
+        'description' => t('The HeaderData class was not found. Install the module with Composer with composer require pantheon-systems/smart_content_cdn .'),
```
Comment (RobLoach) [inline smart_content_cdn.install]:
``` suggestion
        'description' => t('Install the module with Composer with composer require pantheon-systems/smart_content_cdn .'),
```

**Review by jazzsequence — APPROVED** [review]
LGTM! My only question would be if there was a way to wrap `composer require pantheon-systems/smart_content_cdn` in `<code>` tags or something so it stands out a bit more. Other than that :shipit:

File: smart_content_cdn.install [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ */
+function smart_content_cdn_requirements($phase) {
+  $requirements = [];
+
+  if ($phase == 'install') {
+    // Make sure the HeaderData class is available.
+    if (!class_exists('Pantheon\EI\HeaderData')) {
+      $requirements['smart_content_cdn'] = [
+        'title' => t('Smart Content CDN'),
+        'description' => t('Install the module with Composer with composer require pantheon-systems/smart_content_cdn .'),
```
Comment (RobLoach) [inline smart_content_cdn.install]:
``` suggestion
        'description' => t('Install the module with Composer with <code>composer require pantheon-systems/smart_content_cdn</code>.'),
```

**Pull request merged**